src/commands/math/commands.js

```diff
@@ -369,11 +373,11 @@ var SummationNotation = P(MathCommand, function(_, super_) {
   };
 });
 
-LatexCmds['∑'] =
+LatexCmds['?'] =
 LatexCmds.sum =
 LatexCmds.summation = bind(SummationNotation,'\\sum ','&sum;');
 
-LatexCmds['∏'] =
+LatexCmds['?'] =
```

Comment:
I will reconvert the file back to UNICODE (I had to change initially because of the bomb causing the minifying to fail) and commit if it works.